spryker/nginx/usr/local/share/env/25-framework

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+export WEB_INCLUDES=${WEB_INCLUDES:-default-* yves-*}
+export ZED_WEB_INCLUDES=${ZED_WEB_INCLUDES:-default-01-site_ssl default-05-custom_scheme_flags default-05-site_healthcheck default-06-site_redirect_to_https default-10-base default-20-rewriteapp default-30-phpfpm default-50-disable_further_php zed-*}
```

Comment:
Just based on the previous discussion regarding choosing the imports. I can update to be `default-* zed-*`